dockerfiles/Dockerfile-toolchain

```diff
@@ -64,15 +64,21 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o /tmp/rust_install.sh
 RUN chmod +x /tmp/rust_install.sh
 RUN /tmp/rust_install.sh -y
-RUN export PATH="$PATH:$HOME/.cargo/bin"
-RUN echo 'export PATH="$PATH:$HOME/.cargo/bin"' >> /home/opam/.bashrc
```

Comment:
I think this line is worthwhile in addition to the ENV line, but with the same comment that it should be a hardcoded /home/opam path instead of $HOME.